doc/classes/CanvasItem.xml

```diff
@@ -494,7 +494,7 @@
 				Returns the viewport's boundaries as a [Rect2].
 			</description>
 		</method>
-		<method name="get_viewport_transform" qualifiers="const">
+		<method name="get_viewport_transform" qualifiers="const" deprecated="Access canvas and viewport instead to calculate the transform.">
```

Comment:
Assuming the following is correct:
```suggestion
		<method name="get_viewport_transform" qualifiers="const" deprecated="Use [method get_canvas_transform] and [method get_viewport_transform] to calculate the transform, instead.">
```

Reply:
Good idea to explicitly name the necessary transforms, thanks!